packages/metablocks/src/accounts/user.nft.ts

```diff
@@ -0,0 +1,51 @@
+import * as anchor from "@project-serum/anchor";
+import { MetaBlocks } from "../types/meta_blocks";
+import { UserNft } from "../types/types";
+import { camelToSnakeCaseArrayObject, setBlockMetadata } from "./utils";
+
+const getRawUserNfts = async (program: anchor.Program<MetaBlocks>) => {
+  return await program.account.userNft.all();
+};
+
+const getAllUserNfts = async (
+  program: anchor.Program<MetaBlocks>
+): Promise<Array<UserNft>> => {
+  const userNftAccounts = await getRawUserNfts(program);
+
+  const userNfts: Array<UserNft> = [];
+
+  for (const userNftAccount of userNftAccounts) {
```

Comment:
`for`-> `map` 

![bitmoji](https://sdk.bitmoji.com/render/panel/b182cbdd-69b0-445d-86d3-e2bab7f33cd1-18ee6f40-00b3-40e0-87de-e4f7808f57d4-v1.png?transparent=1&palette=1&width=246)